virtual_kubelet/node/node.go

```
@@ -146,8 +143,9 @@ type NodeController struct { //nolint:revive
 	errMu  sync.Mutex
 	err    error
 
-	nodePingController *nodePingController
-	pingTimeout        *time.Duration
+	// this is only usefully when the vk is deploy beside the node
+	//nodePingController *nodePingController
+	pingTimeout *time.Duration
```

golangci-lint (1.62.2)

field `pingTimeout` is unused

(unused)